test/integrationConfiguration.test.js

```diff
@@ -0,0 +1,217 @@
+        assert.strictEqual(actual.technicalAccount.privateKey, expected.technicalAccount.privateKey);
+
+
+        process.env.ASSET_COMPUTE_INTEGRATION_FILE_PATH = yamlFile;
+        actual = await IntegrationConfiguration.getConfiguration();
+
+        assert.strictEqual(actual.metascopes[0], expected.metascopes[0]);
+        assert.strictEqual(actual.metascopes[1], expected.metascopes[1]);
+        assert.strictEqual(actual.metascopes[2], expected.metascopes[2]);
+
+        assert.strictEqual(actual.technicalAccount.id, expected.technicalAccount.id);
+        assert.strictEqual(actual.technicalAccount.org, expected.technicalAccount.org);
+        assert.strictEqual(actual.technicalAccount.clientId, expected.technicalAccount.clientId);
+        assert.strictEqual(actual.technicalAccount.clientSecret, expected.technicalAccount.clientSecret);
+        assert.strictEqual(actual.technicalAccount.privateKey, expected.technicalAccount.privateKey);
+    });
+
+    it('Input file is JSON', async () => {
+
+        expected.technicalAccount.privateKey = "PRIVATE KEY FILE\n";
+        let actual = await IntegrationConfiguration.getConfiguration(jsonFile, privateKeyFile);
+
+        assert.strictEqual(actual.metascopes[0], expected.metascopes[0]);
+        assert.strictEqual(actual.metascopes[1], expected.metascopes[1]);
+        assert.strictEqual(actual.metascopes[2], expected.metascopes[2]);
+
+        assert.strictEqual(actual.technicalAccount.id, expected.technicalAccount.id);
+        assert.strictEqual(actual.technicalAccount.org, expected.technicalAccount.org);
+        assert.strictEqual(actual.technicalAccount.clientId, expected.technicalAccount.clientId);
+        assert.strictEqual(actual.technicalAccount.clientSecret, expected.technicalAccount.clientSecret);
+        assert.strictEqual(actual.technicalAccount.privateKey, expected.technicalAccount.privateKey);
+
+
+        process.env.ASSET_COMPUTE_INTEGRATION_FILE_PATH = jsonFile;
+        process.env.ASSET_COMPUTE_PRIVATE_KEY_FILE_PATH = privateKeyFile;
+        actual = await IntegrationConfiguration.getConfiguration();
+
+        assert.strictEqual(actual.metascopes[0], expected.metascopes[0]);
+        assert.strictEqual(actual.metascopes[1], expected.metascopes[1]);
+        assert.strictEqual(actual.metascopes[2], expected.metascopes[2]);
+
+        assert.strictEqual(actual.technicalAccount.id, expected.technicalAccount.id);
+        assert.strictEqual(actual.technicalAccount.org, expected.technicalAccount.org);
+        assert.strictEqual(actual.technicalAccount.clientId, expected.technicalAccount.clientId);
+        assert.strictEqual(actual.technicalAccount.clientSecret, expected.technicalAccount.clientSecret);
+        assert.strictEqual(actual.technicalAccount.privateKey, expected.technicalAccount.privateKey);
+
+
+        process.env.ASSET_COMPUTE_PRIVATE_KEY_FILE_PATH = privateKeyFile;
```

Comment:
Put this in another test (a separate it(...). This will ensure there are no leftovers in actual from the test above. Leftovers render the test inconclusive: the test may pass because of old actual values (for instance, if loading from the env variable set values fails silently).

The test title could be `it('Input file is JSON from env variable')`.